java-util/src/main/java/io/druid/java/util/common/StringUtils.java

```diff
@@ -66,8 +67,12 @@ public static int estimatedBinaryLengthAsUTF8(String value)
     return string == null ? EMPTY_BYTES : toUtf8(string);
   }
 
+  @Nullable
```

Comment:
To not make some generic code less safe, suggested to add separate `fromUtf8Nullable()` and `toUtf8Nullable()` methods